src/app/store.ts

```diff
@@ -1,10 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
+import { TypedUseSelectorHook, useSelector } from 'react-redux';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
The function `combineSlices` is not a standard function in Redux Toolkit. It should be `combineReducers` to correctly combine the reducers from the slices.